iocage/lib/JailConfigAddresses.py

```diff
@@ -0,0 +1,89 @@
+class AddressSet(set):
+
+    def __init__(self, jail_config=None, property_name="ip4_address"):
+        self.jail_config = jail_config
+        set.__init__(self)
+        object.__setattr__(self, 'property_name', property_name)
+
+    def add(self, value, notify=True):
+        set.add(self, value)
+        if notify:
+            self.__notify()
+
+    def remove(self, value, notify=True):
+        set.remove(self, value)
+        if notify:
+            self.__notify()
+
+    def __notify(self):
+        try:
+            self.jail_config.update_special_property(self.property_name)
+        except:
```

Comment:
The reason behind this: We don't care if JailConfig fails updating a special property (such as `ip4_addr`). I left a note that we can remove the try/catch after we added unit tests for JailConfig.